src/uhs/atomizer/shard/shard.hpp

```diff
@@ -61,7 +61,7 @@ namespace cbdc::shard {
         /// \return true if the shard successfully digested the block. False if the block height is not contiguous.
         auto digest_block(const cbdc::atomizer::block& blk) -> bool;
 
-        /// Returns the height of the the most recently digested block.
+        /// Returns the height of the most recently digested block.
```

Comment:
This isn't strictly a spelling fix, but old fashioned spell checkers had support for this flag, so I've been prototyping it.

Reply:
Still a typo and a good opportunity to improve readability. Thank you for flagging it.